config/routes.rb

```diff
@@ -80,6 +80,7 @@
         resources :commodities, only: [:index] do
           get :search, on: :collection
         end
+        resources :commodities_from_contexts, only: [:index]
```

Comment:
would it be possible to take it out of the /dashboards namespace and just call it 'commodities'? It is not dashboards specific, and in fact we could even include this in the public API.